### PR TITLE
eshost --add: support inclusion of --tags (see details for examples)

### PR DESCRIPTION
**Pull request by rwaldron**
- eshost --add ch ch `which ch` --tags latest
- eshost --add ch ch `which ch` --tags latest,greatest

Tests will follow in another PR which contains all the tests that I've written 

Signed-off-by: Rick Waldron <waldron.rick@gmail.com>

### CODE REVIEW

File: lib/host-manager.js [hunk 1]
```diff
@@ -50,11 +50,16 @@ exports.add = function add(config, name, type, path, args) {
...
   }
 
-  if(path && !Path.isAbsolute(path)) {
+  if (path && !Path.isAbsolute(path)) {
     path = Path.join(process.cwd(), path);
   }
 
-  config.hosts[name] = { type, path, args };
+  if (tags) {
+    if (typeof tags === 'string') {
```
Comment (rwaldron) [inline lib/host-manager.js]: Meh. I suppose these could be combined into a single expression

File: lib/host-manager.js [hunk 2]
```diff
@@ -50,11 +50,16 @@ exports.add = function add(config, name, type, path, args) {
...
+      tags = [tags];
```
Comment (dilijev) [inline lib/host-manager.js]: Isn't this already taken care of with `hostTags = argv.tags.split(',');` (and then `hostTags` is passed in for the `tags` param to this function?

Comment (rwaldron) [inline lib/host-manager.js]: Indeed it is. I will correct this.

**Comment by rwaldron** [comment]
Assuming the "approval" is still valid, I'd like to include this in tested features, so I'm landing it.

**Pull request merged**